uber/utils.py

```diff
@@ -516,3 +516,15 @@ def __exit__(self, type, value, traceback):
     @staticmethod
     def _clear_cache():
         threadlocal.clear()
+
+def normalize_email(address):
```

Comment:
This is awesome! But since this only works on gmail domains, can we change the name of the function to reflect that? That way no-one tries to use it later thinking it does something else.